tslint.json

```diff
@@ -11,6 +11,7 @@
         "indent": [true,
             "spaces"
         ],
+        "inline-comment-spacing": true,

```

Comment:
Maybe we should have a separate rule for `"inline-callExpressionArgument-comment-spacing"`? Or maybe name it analogously to `"boolean-trivia"`? eg: `"boolean-trivia-spacing"`?